### PR TITLE
Replace some setters and getters from Camera

### PR DESCRIPTION
**Pull request by leotrs**
Title. Also turn `ThreeDCamera.frame_center` into a property. This is the only member I changed into a property since I don't want to clash with the `attrs` branch. I had to turn it into a property otherwise the class won't work.

### CODE REVIEW

**Comment by leotrs** [comment]
This is a test comment please ignore

**Comment by leotrs** [comment]
This is a test comment please ignore

Depends on #338

File: manim/camera/three_d_camera.py [hunk 1]
```diff
@@ -46,11 +45,19 @@ def __init__(self, *args, **kwargs):
         self.distance_tracker = ValueTracker(self.distance)
         self.gamma_tracker = ValueTracker(self.gamma)
         self.light_source = Point(self.light_source_start_point)
-        self.frame_center = Point(self.frame_center)
+        self._frame_center = Point(kwargs.get("frame_center", ORIGIN))
```
[the remark this answers is not in the document]
Comment (leotrs) [reply manim/camera/three_d_camera.py:48]: @PgBiel I suggest we merge this as is so development can move forward and revisit this in the future in the attrs branch.

[the remark this answers is not in the document]
Comment (leotrs) [reply manim/camera/three_d_camera.py:48]: @PgBiel any thoughts here?

**Review by huguesdevimeux — APPROVED** [review]
SGTM

**Pull request merged**